src/current/v23.2/connection-pooling.md

```diff
@@ -37,7 +37,11 @@ These are Cockroach Labs recommendations for settings common to most connection
 
 ### Set the maximum lifetime of connections
 
-The maximum lifetime of a connection should be set to no less than 5 minutes and no more than 30 minutes. Applications that connect to {{ site.data.products.serverless }} clusters should set this to 30 minutes, while applications that connect to {{ site.data.products.dedicated }} and {{ site.data.products.core }} clusters should set this to 5 minutes.
+The maximum lifetime of a connection should be set to between 5 and 30 minutes. 30 minutes is the maximum supported connection lifetime supported by {{ site.data.products.dedicated }} and {{ site.data.products.serverless }}, and client connections are likely to be reset after 30 minutes, causing a disruption to applications.
```

Comment:
> client connections are likely to be reset after 30 minutes, causing a disruption to applications.

that language is a bit more aggressive than what we need, i think. we can say:

> during node shutdown and restart, client connections can be reset after 30 minutes, causing a disruption to applications.

Reply:
Done